single.php

```diff
@@ -23,30 +23,74 @@
 
 		the_post();
 
-		// Remove support for the global hero template part. Intended for pages, primarily.
-		// get_template_part( 'templates-global/hero' ); .
-
 		get_template_part( 'templates-global/global-banner' );
+		get_template_part( 'templates-global/story-hero' );
+
+		?>
+
+		<article <?php post_class(); ?> id="post-<?php the_ID(); ?>">
+
+			<?php
+			// TODO: Identify how we will add and control the social media "intent to repost" icons. Sample markup follows.
+
+			// <header class="entry-header">
+			// <div class="row" id="social-media">
+			// <nav class="nav flaot-left" aria-label="Social Media">
+			// <a class="nav-link text-maroon" href="#"><span title="Facebook" class="fab fa-facebook-square"></span></a>
+			// <a class="nav-link text-maroon" href="#"><span title="Twitter" class="fab fa-twitter-square"></span></a>
+			// <a class="nav-link text-maroon" href="#"><span title="LinkedIn" class="fab fa-linkedin"></span></a>
+			// </ul>
+			// </div>
+
+			the_content();
 
-		echo '<div class="container">';
-		echo '<div class="row">';
-		echo '<div class="col">';
+			$author_name = get_field( 'name' );
+			$author_title = get_field( 'title' );
+			$author_email = get_field( 'email' );
+			$author_phone = get_field( 'phone' );
+			if ( $author_name || $author_title || $author_email || $author_phone ) {
+				echo '<div class="author_info">';
+				if ( $author_name ) {
+					echo '<h4><span class="highlight-gold">' . $author_name . '</span></h4>';
+				}
+				if ( $author_title ) {
+					echo '<p>' . $author_title . '</p>';
+				}
+				if ( $author_email || $author_phone ) {
+					echo '<p>';
+					if ( $author_email ) {
+						echo '<span class="fas fa-envelope-square"></span><a href="mailto:' . $author_email . '">' . $author_email . '</a>';
+					}
+					echo '</br>';
+					if ( $author_phone ) {
+						echo '<span class="fas fa-phone-square"></span><a href="tel:' . $author_phone . '">' . $author_phone . '</a>';
```

Comment:
Note that the ACF field for author phone is a Number field (which is great by the way!) That means the phone value needs to be formatted here. The UDS standards specifies telephone numbers must use dashes, not parenthesis: xxx-xxx-xxxx